### PR TITLE
New tutorial "Protein ID using OpenMS" and major revision of "Protein ID using SG/PS"

### PR DESCRIPTION
**Pull request by Stortebecker**
Hi!

I added a new tutorial which provides an alternative to the SG/PS tutorial. 

Also, I updated the SG/PS tutorial to fit the current version of the Galaxy wrappers, which had evolved quite a bit. I deleted a lot of comments which I found to be more confusing than helping. 

The ntails and secretome-prediction tutorials listed below are unchanged (see the rich diff), I do not know why they are listed as "changed files".

Cheers, Flo

### CODE REVIEW

File: topics/proteomics/metadata.yaml [hunk 1]
```diff
@@ -123,13 +123,36 @@ material:
       - "Protein identification from LC-MS/MS raw files."
     time_estimation: "45min"
     key_points:
-      - "LC-MS/MS raw files have to be converted before using GalaxyP on most GalaxyP servers."
+      - "LC-MS/MS raw files have to be converted to mgf/mzML before using GalaxyP on most GalaxyP servers."
```
Comment (blankclemens) [inline topics/proteomics/metadata.yaml]: The specific mentioning of GalaxyP without explanation might be a bit confusing? What is a "GalaxyP" server, what is "using GalaxyP"? It's all Galaxy after all. Would just say: "LC-MS/MS raw files have to be converted to mgf/mzML for further analysis."

File: topics/proteomics/tutorials/protein-id-oms/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+To find out the peptide sequences, the MS2 spectrum is compared to a theoretical spectrum generated from a protein database. This step is called peptide-to-spectrum (also: spectrum-to-sequence) matching. Accordingly, a peptide that is successfully matched to a sequence is termed PSM (Peptide-Spectrum-Match). There can be multiple PSMs per peptide, if the peptide was fragmented several times.
+
+Different peptide search engines have been developed to fulfill the matching procedure. Here, we will use the search engine [X!Tandem](https://www.ncbi.nlm.nih.gov/pubmed/14976030). OpenMS provides "adapters" (wrappers) for several other peptide search engines, like MSGF+ or OMSSA. You may replace the XTandemAdapter by another search engine of your choice. 
+
+> ### {% icon hands_on %} Hands-On: Peptide Identification
+>
+> 1. Copy the prepared protein database from the tutorial [Database Handling](../database-handling/tutorial.md) into your current history by using the multiple history view or upload the ready-made database from this [link](https://zenodo.org/record/892005/files/Human_database_including_decoys_%28cRAP_and_Mycoplasma_added%29.fasta).
```
Comment (blankclemens) [inline topics/proteomics/tutorials/protein-id-oms/tutorial.md]: Did this link work in your testing instance? Travis complains, but I don't know what's the right approach for internal links. @bebatut

Comment (shiltemann) [reply topics/proteomics/tutorials/protein-id-oms/tutorial.md]: jekyll compiles the markdown files to html, so you need to change the extension here to `.html`

Comment (Stortebecker) [reply topics/proteomics/tutorials/protein-id-oms/tutorial.md]: Thanks for the tip @shiltemann . This applies to several of my tutorials. I should change that.

**Pull request merged**